bin/plugin/commands/since.js

```diff
@@ -77,21 +77,29 @@ exports.handler = async ( opt ) => {
 			ignore,
 		} );
 
-		const regexp = /(@since\s+)n\.e\.x\.t/g;
+		const regexps = [
+			/(@since\s+)n\.e\.x\.t/g,
+			/('[^']+?)n\.e\.x\.t(?=')/g,
```

Comment:
Maybe this is better instead:

```suggestion
			/('[^']*?)n\.e\.x\.t(?=')/g,
```

This would also match strings like `'n.e.x.t'` and not just those that start with the plugin name.